packages/html/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # @kitajs/html
 
+## 4.0.0-next.3
+
+### Minor Changes
+
+- [`bab34f8`](https://github.com/kitajs/html/pull/166/commits/bab34f87a9043b117b528ca3789dcba47fb8ca73)
```

Comment:
Our changeset is generated automatically. You just need to run `pnpm changeset` at the project root... Can you reset this change?

Reply:
Sure, good to know! Thanks